### PR TITLE
Issue #163: Centered the edit profile page and made the buttons consistent with the page

### PR DESCRIPTION
**Pull request by daminisatya**
selection_044

selection_043

### CODE REVIEW

**Comment by daminisatya** [comment]
@tonythomas01 Will change the mentioned things. I thought the same about the <div class="mt40"> </div> but dint wanted to disturb the existing code.

**Comment by tonythomas01** [comment]
(any update on this one) ?

File: systers_portal/templates/users/edit_profile.html [hunk 1]
```diff
@@ -12,7 +11,7 @@
 {% block content %}
   <div class="mt40"></div>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/users/edit_profile.html]: delete.

File: systers_portal/templates/users/edit_profile.html [hunk 2]
```diff
@@ -12,7 +11,7 @@
...
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-md-6 col-md-offset-3">
```
Comment (tapasweni-pathak) [inline systers_portal/templates/users/edit_profile.html]: As Tony said ```col-md-4 col-md-offset-4``` is better.

File: systers_portal/templates/users/snippets/profile.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
     {% if user == systersuser.user or user.is_superuser %}
       <div class="pull-right">
         <a href="{% url 'user_profile' systersuser.user.username %}"
-           role="button" class="btn btn-primary">Edit profile</a>
+           role="button" class="btn btn-default mr4">Edit profile</a>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/users/snippets/profile.html]: why this? :) 
does the default one suits more than primary here for you?

**Comment by tapasweni-pathak** [comment]
Did the few corrections required, squashed and merged. Thanks for the work!

**Pull request closed without merging**